storey/aggregations.py

```diff
@@ -123,13 +127,35 @@ async def _do(self, event):
             if isinstance(self._emit_policy, EmitEveryEvent):
                 await self._emit_event(key, event)
             elif isinstance(self._emit_policy, EmitAfterMaxEvent):
-                self._events_in_batch[key] = self._events_in_batch.get(key, 0) + 1
-                if self._events_in_batch[key] == self._emit_policy.max_events:
-                    await self._emit_event(key, event)
-                    self._events_in_batch[key] = 0
+                if key in self._events_in_batch:
+                    self._events_in_batch[key]['counter'] += 1
+                else:
+                    self._events_in_batch[key] = {}
+                    self._events_in_batch[key]['counter'] = 1
+                    self._events_in_batch[key]['time'] = time.monotonic()
```

Comment:
Can we just create the dictionary and then put it in the outer dictionary? To save on looking up `self._events_in_batch[key]` over and over.